packages/ui/src/components/styled-link.tsx

```diff
@@ -1,27 +1,86 @@
-import { RefAttributes } from 'react'
-import type { LinkProps } from 'react-router-dom'
+import { MouseEvent, RefAttributes } from 'react'
+import type { LinkProps as LinkBaseProps } from 'react-router-dom'
 
 import { useRouterContext } from '@/context'
 import { cn } from '@utils/cn'
 import { cva, VariantProps } from 'class-variance-authority'
 
-const linkVariants = cva(
-  'whitespace-nowrap decoration-transparent decoration-1 underline-offset-4 transition-colors duration-200',
-  {
-    variants: {
-      variant: {
-        default: 'hover:decoration-foreground-1 text-cn-foreground-1',
-        secondary: 'hover:decoration-foreground-4 text-cn-foreground-2',
-        accent: 'hover:decoration-foreground-accent text-cn-foreground-accent'
-      }
+import { Icon, IconProps } from './icon'
+
+export const linkVariants = cva('cn-link', {
+  variants: {
+    variant: {
+      default: 'cn-link-default',
+      secondary: 'cn-link-secondary'
+    },
+    size: {
+      default: '',
+      sm: 'cn-link-sm'
     }
   }
-)
+})
```

Comment:
Looks like we don't have defaults specifically called out for `size` and `variant`.

```suggestion
export const linkVariants = cva('cn-link', {
  variants: {
    variant: {
      default: 'cn-link-default',
      secondary: 'cn-link-secondary'
    },
    size: {
      default: '',
      sm: 'cn-link-sm'
    }
  },
  defaultVariants: {
    variant: 'default'
    size: 'default'
  }
})
```

Reply:
Added